src/core/agent/services/keriaNotificationService.ts

```diff
@@ -1072,6 +1072,12 @@ class KeriaNotificationService extends AgentService {
               });
             }
           }
+
+          await this.ipexCommunications.createLinkedIpexMessageRecord(
+            admitExchange,
+            ConnectionHistoryType.CREDENTIAL_ADMITTED
```

Comment:
Admitting a credential is receiving a credential in our case. Please read the story.

This is what the PR should do:
- Wherever we are creating records for `ConnectionHistoryType.CREDENTIAL_ISSUANCE` in ipexCommuncationService,
- Move it to keriaNotificationService when the long running admit finishes.

And remove this new `CREDENTIAL_ADMITTED`